Sources/Purchasing/Purchases/PurchasesType.swift

```diff
@@ -277,13 +277,15 @@ public protocol PurchasesType: AnyObject {
      * - Important: Call this method when a user has decided to purchase a product.
      * Only call this in direct response to user input.
      *
-     * From here ``Purchases`` will handle the purchase with `StoreKit` and call the ``PurchaseCompletedBlock``.
+     * From here ``Purchases`` will handle the purchase with `StoreKit` and return ``PurchaseResultData``.
      *
-     * - Note: You do not need to finish the transaction yourself in the completion callback, ``Purchases`` will
+     * - Note: You do not need to finish the transaction yourself after this, ``Purchases`` will
      * handle this for you.
      *
      * - Parameter product: The ``StoreProduct`` the user intends to purchase.
      *
+     * If the purchase was successful there will be a ``StoreTransaction`` and a ``CustomerInfo``.
```

Comment:
Not sure if this is true!! for async API

Reply:
Yup that's part of the tuple 👍🏻 maybe put this and the next line about `userCancelled` inside of `- Returns:` blocks?